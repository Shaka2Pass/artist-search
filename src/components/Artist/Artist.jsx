//Artist detail page (presentational component) with all releases.
//Paging through releases
//http://musicbrainz.org/ws/2/release?artist=<ARTIST_ID>&fmt=json

import React from 'React';
import PropTypes from 'prop-types';

const Artist = () => (
  <h4>{name}</h4>
);

Artist.propTypes = {
  name: PropTypes.isRequired
};

export default Artist;

