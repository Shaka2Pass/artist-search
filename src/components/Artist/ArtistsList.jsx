import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';

const ArtistList = ({name}) => (

);

ArtistsList.propTypes = {
  name: PropTypes.string.isRequired
};

export default Artists