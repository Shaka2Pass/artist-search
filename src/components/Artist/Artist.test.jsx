import React from 'react';
import { shallow } from 'enzyme';
import Artist from './Artist';

describe('It describes the Artist component', () => {
  it('matches a snapshot of Artist.jsx', () => {
    const wrapper = shallow(<Artist name="U2"/>);
    expect(wrapper).toMatchSnapshot();
  });
});

