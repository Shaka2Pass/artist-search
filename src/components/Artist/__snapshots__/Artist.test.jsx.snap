// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It describes the Artist component matches a snapshot of Artist.jsx 1`] = `<h4 />`;
